aws-v4.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+gems = Dir['gems/*']
+gems.each do |gem_path|
+  $LOAD_PATH.unshift(File.expand_path('lib', gem_path))
+end
+# TODO: This is slow - we should autoload but it requires a complete
+# code generated list of services like aws-sdk-resources.
```

Comment:
I think we should potentially write out a services.json sort of file during codegeneration that we could use here.  The autoload is highly preferable.

Reply:
yeah. out of scope right now ..